### DISCUSSION TITLE
Third party using focus trap can we exit?

### DISCUSSION THREAD

**Discussion by philipbradbury1** [focus-trap/focus-trap#1123] [Q&A]
Looking to see if there is a possible solution for this issue I have. I have built a site on top of a template system. That system has a modal that uses focus-trap. In that modal we have used lightgallery. The issue is when light gallery opens the focus is stuck on the modal. Is there a way to cancel the focus given we don't have access to the original focus trap object?

**Comment by stefcameron** [comment]
@philipbradbury1 Could you elaborate more on what you mean by focus being "stuck" on the modal? Do you mean it's on the modal itself instead of being on a focusable/tabbable component _inside_ of it and you can't click on anything because the focus keeps coming back to the modal itself? Focus trap is supposed to trap focus, after all, so how is it "stuck"? And what is "lightgallery"?
